client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -20,6 +20,16 @@ export default function OnboardingEdit() {
 		( e: React.ChangeEvent< HTMLInputElement > ) => setSiteTitle( e.target.value ),
 		[ setSiteTitle ]
 	);
+	const handleSiteTypeChange = useCallback(
```

Comment:
While it's fine to prefix event handlers with `handle`, I think it's also a bit redundant -- we could rename this to

```suggestion
	const updateSiteType = useCallback(
```

I think that would sound a bit more semantic.